packages/es-ingest/src/main/resources/application.properties

```diff
@@ -9,7 +9,35 @@ logging.pattern.console=%d{yyyy-MM-dd HH:mm:ss} [%level] %msg [%c:%L]%n
 ########################################################################################################################
 terarium.elasticsearch.url=https://elasticsearch.staging.terarium.ai:443
 terarium.elasticsearch.index.prefix=tds
-terarium.elasticsearch.index.suffix=tera_1.0
-terarium.elasticsearch.index.searchable-document-root=searchable_document
-terarium.elasticsearch.index.searchable-model-root=searchable_model
-management.health.elasticsearch.enabled=false
+terarium.elasticsearch.index.suffix=tera_2.0
+terarium.elasticsearch.auth-enabled=true
+terarium.elasticsearch.username=${es-username}
+terarium.elasticsearch.password=${es-password}
+
+########################################################################################################################
+# aws credentials configuration
+########################################################################################################################
+aws-access-key-id=admin
+aws-secret-access-key=admin123
+aws-url=
+terarium.file-storage-s3-bucket-name=askem-staging-data-service
+terarium.amazon.credential.access-key=${aws-access-key-id}
+terarium.amazon.credential.secret-key=${aws-secret-access-key}
+terarium.amazon.s3.region=us-east-1
+terarium.amazon.s3.url=${aws-url}
+
+########################################################################################################################
+# Ingest configuration
+########################################################################################################################
+terarium.ingestParams[0].name="Model Dataset"
+terarium.ingestParams[0].inputDir=/home/kbirk/Downloads/model
```

Comment:
You should add those to the readme to know that those needs to be changed when running the application

Reply:
It's all in the README.